cin_validator/utils.py

```diff
@@ -21,13 +30,14 @@ def make_census_period(reference_date):
     # reference_date is a pandas series. Get it's value as a string by indexing the series' values array.
     reference_date = reference_date.values[0]
 
+    #  Try/except to allow for different datetime formats
+
     # the ReferenceDate value is always the collection_end date
-    collection_end = pd.to_datetime(reference_date, format="%d/%m/%Y")
+    collection_end = make_date(reference_date)
+
     # the collection start is the 1st of April of the previous year.
     collection_start = (
-        pd.to_datetime(reference_date, format="%d/%m/%Y")
-        - pd.DateOffset(years=1)
-        + pd.DateOffset(days=1)
+        make_date(reference_date) - pd.DateOffset(years=1) + pd.DateOffset(days=1)
```

Comment:
This no longer needs to be surrounded by brackets. the make_date statement can exist without the brackets around it on lines 39 and 41.

Reply:
black actually automatically adds the brackets, it's to keep line length within 79 characters believe